matrix/tests_matrix.py

```diff
@@ -0,0 +1,147 @@
+import unittest
+
+import pytest
+from matrix import Matrix
+
+
+@pytest.fixture(scope="module")
+def db_class(request):
+    # set a class attribute on the invoking test context
+    request.cls.db = Matrix()
+
+
+class TestMatrix(unittest.TestCase):
```

Comment:
Look at the purpose of unitest \ testcase \ testsuite
Sometimes its better to separate the test to several cases (for example, 1 case for testing creating function, 1 for operations, etc.)

Reply:
Do you mean to seperate to different classes?

Reply:
Yes